atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -197,7 +199,9 @@ long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
             } else {
                 SweepBatchConfig lastBatchConfig = getAdjustedBatchConfig();
                 log.error("The background sweep job failed unexpectedly with batch config {}"
```

Comment:
Agreed. Downgrading to warn here.

Should I also downgrade on the logging above? I would say it's not an error, as in, sweep will recover properly from that situation. I would probably downgrade that to info.